src/lib/db/feature-environment-store.ts

```diff
@@ -258,6 +264,7 @@ export class FeatureEnvironmentStore implements IFeatureEnvironmentStore {
     async connectFeatureToEnvironmentsForProject(
         featureName: string,
         projectId: string,
+        enabledIn: { [environment: string]: boolean } = {},
```

Comment:
This adds support to pass the desired state of the feature in that environment, which is required for this assertion: https://github.com/Unleash/unleash/pull/2447/files#diff-696f6242d5d467fdce764b004ca9dda90678e55f5051153dd5196d21a9c49e03R463

We were always setting the relationship to `enabled: false` https://github.com/Unleash/unleash/pull/2447/files#diff-299d69b93e5da7b84238a0d5564d36d4129bee60cfce0e0e84511a00375e04a7L271